included_software.md

```diff
@@ -11,7 +11,7 @@ The specific patch versions included will depend on when the image was last buil
 * Ruby - `RUBY_VERSION`, `.ruby-version`
   * 2.7.2 (default)
   * 2.6.6
-  * Any version that `rvm` can install.
+  * Any version >= 2.4.0 that `rvm` can install.
```

Comment:
@verythorough, I do like the brevity here and think it's pretty clear.

But, if we're looking for consistency, we try to use "earlier" and "later" wording with versions. So, something like the following may be appropriate:
```suggestion
  * Any version 2.4.0 or later that `rvm` can install.
```